tekton/images/kubectl/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Copyright 2019 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+FROM alpine:3.10
+LABEL maintainer "Andrea Frittoli <andrea.frittoli@gmail.com>"
```

Comment:
nit: in the future, we may want to use `The Tekton Authors <tekton-dev@google…>` as maintainer(s) of the `Dockerfile` :stuck_out_tongue: (like done [here](https://github.com/tektoncd/plumbing/blob/master/prow/images/test-runner/Dockerfile#L16))

Reply:
Nice catch, thank you! :)

Reply:
Done